test/src/test/java/hudson/logging/LogRecordManagerRealTest.java

```diff
@@ -0,0 +1,34 @@
+package hudson.logging;
+
+import java.util.List;
+import java.util.logging.LogRecord;
+import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RealJenkinsRule;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+public class LogRecordManagerRealTest {
+
+    @Rule
+    public RealJenkinsRule rr = new RealJenkinsRule();
+
+    @Test
+    public void logRecordsArePresentOnController() throws Throwable {
+        rr.then(new LogRecordsArePresent());
+    }
+
+    private static class LogRecordsArePresent implements RealJenkinsRule.Step {
```

Comment:
FYI this is conventionally done via a `static` method and a method reference. See Javadoc.

Reply:
ah cool feel free to adjust, for some reason by IDE wouldn't download javadoc while I was writing it so I just searched the org for examples 😢